generator/toyota/lexus_ct200h_2018_pt.dbc

```diff
@@ -0,0 +1,37 @@
+CM_ "IMPORT _toyota_2017.dbc"
+CM_ "IMPORT _comma.dbc"
+
+BO_ 548 BRAKE_MODULE: 8 XXX
+ SG_ BRAKE_PRESSURE : 0|9@0+ (1,0) [0|511] "" XXX
+ SG_ BRAKE_POSITION : 16|9@0+ (1,0) [0|511] "" XXX
+ SG_ BRAKE_PRESSED : 37|1@0+ (1,0) [0|1] "" XXX
```

Comment:
@thomaspich , has been a while... but while doing some clean up I've noticed that the LEXUS_CTH is the only car to have the brake_pressed on this combination of message address and bit. DO you happen to have a recent LEXUS_CTH route you can share?